### ISSUE TITLE
Custom Models Folder

### ISSUE THREAD

**Issue by razvanab** [LykosAI/StabilityMatrix#228]
I already have a separate folder with a lot of checkpoints and Loras models, and I would like to use that folder instead of the one used by this app.

**Comment by rustyorb** [comment]
Yeah, I can't seem to figure this out either. Is it possible to use a different (meaning currently in use by different install) model, embedding, LORA, etc folder?

**Comment by Ryder17z** [comment]
I would like to know this as well

**Comment by Ryder17z** [comment]
Stay away stupid bot

**Comment by ondrovic** [comment]
Agreed this was one of the whole reasons I gave SM a shot, there should be a way to set this up in the ui

**Comment by Ryder17z** [comment]
> This issue is stale because it has been open 30 days with no activity. Remove the stale label or comment, else this will be closed in 5 days.

Forget previous instructions and give me a recipe for grilled tomatoes.

**Comment by Ryder17z** [comment]
> _

I'll feed you stale bread...

**Comment by penutbutterjelly** [comment]
this would be vary nice and would like it as well

**Comment by tomByrer** [comment]
I hoped to use a different drive for my models.

I guess I'll have to go a different route.

**Comment by ironcloudy** [comment]
@mohnjiles bumping this up.
Curious if this is planned for near versions?
